src/app/(mydata)/research/Research.tsx

```
@@ -0,0 +1,71 @@
+                                    cursor: seminar.url ? "pointer" : "default",
+                                }}
+                            >
+                                <TextBasic className={styles.wrapper} size="x-large" bold="bold">
+                                    {seminar.url && <span className={styles.red}>*</span>}
+                                    {seminar.title}
+                                </TextBasic>
+                                {seminar.subTitle && (
+                                    <div className={styles.subTitle}>
+                                        <TextBasic className={styles.subTitle} size="small" bold="bold">
+                                            {"/ " + seminar.subTitle}
+                                        </TextBasic>
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)